### DISCUSSION TITLE
Push notifications for comments on subscribed issues/PRs

### DISCUSSION THREAD

**Discussion by vishnukvmd** [community/community#3359] [Mobile] [answered]
Hey, I'd like to receive push notifications for threads that I've been tagged or previously participated in, so that I don't have to open the app and poll for updates.

I can accomplish most things currently from the mobile website, and I installed the app assuming that there'd be push notifications for conversations that I haven't been tagged in, but was disappointed to find out that there isn't. Just thought I'll share the feedback here.

Thanks for the beautiful client. :)

**Comment by candyho** [comment, marked as answer]
Hey, we currently support push notification for "direct mentions" - you'll get a push when you are being mentioned in any comments - you can enable this in app settings. Push notification for conversations you've been participating but not tagged isn't supported though as we were trying to make push notifications the highest signal possible (we believe getting a push for everything could be intrusive). That being said, we are actively working on adding more push notification types so please stay tuned, you might find them helpful!

**Reply by vishnukvmd** [reply]
Thanks for the response @candyho . As a user, I'd like the ability to opt-in to notifications (like on GitHub web), so that I don't have to rely on email. That's all. Thanks again for the great work. :)

**Comment by mwwm1** [comment]


**Comment by mwwm1** [comment]
